src/main/java/com/google/sps/data/GameStage.java

```diff
@@ -138,4 +149,13 @@ public void setContent(String content) {
   public void setQuizKey(String quizKey) {
     this.quizKey = quizKey;
   }
+
+  /**
+   * Sets the game stage's dialogue
+   *
+   * @param dialogue An array of string representing the text that will show up on game stage
+   */
+  public void setDialogue() {
+    dialogue = content.split(";");
+  }
```

Comment:
Now that the split is happening in front end, we could take out these changes?

Reply:
yes